lib/logcat.js

```diff
@@ -4,14 +4,17 @@ import B from 'bluebird';
 
 
 const log = logger.getLogger('Logcat');
+const MAX_BUFFER_SIZE = 10000;
+
 
 class Logcat {
   constructor (opts = {}) {
     this.adb = opts.adb;
     this.debug = opts.debug;
     this.debugTrace = opts.debugTrace;
+    this.maxBufferSize = opts.maxBufferSize || MAX_BUFFER_SIZE;
```

Comment:
Is 10000 a hard limit? Should people be allowed to enter a `opts.maxBufferSize` that is higher?

Reply:
it's the default value. This default has proven to be ok with iOS. 

> Should people be allowed to enter a opts.maxBufferSize that is higher?
Why not. Currently this will always be assigned to the default value, but then we might change the the default value in client drivers if needed.